meals/models.py

```diff
@@ -4,9 +4,9 @@
 from django.contrib.auth.models import User
 
 
-
 class Menu(models.Model):
     name = models.CharField(max_length=100)
+    users = models.ManyToManyField(User, related_name='menus')
```

Comment:
This implicitly creates a UserMenu model for you, which is ok as long as you don't need to customize it.  For the most part you don't need to deal directly with that model.  Instead you can access menus through users like: `user.menus` or the other way around: `menu.users`